datafusion-cli/src/object_storage.rs

```diff
@@ -141,7 +141,13 @@ mod tests {
         assert!(err.to_string().contains("Generic S3 error: Missing region"));
 
         env::set_var("AWS_REGION", "us-east-1");
-        assert!(provider.get_by_url(&Url::from_str(s3).unwrap()).is_ok());
+        let url = Url::from_str(s3).expect("Unable to parse s3 url");
+        let res = provider.get_by_url(&url);
+        let msg = match res {
+            Err(e) => format!("{}", e),
+            Ok(_) => "".to_string()
+        };
+        assert_eq!("".to_string(), msg); // Fail with error message
```

Comment:
I refactored this test to have a better failure message because it was failing for me locally. The fact that CI passed makes me think it's not running there because `datafusion-cli` is excluded from the workspace - I don't know why this is, but would propose we include it.

Reply:
Could you please file a ticket to do so? Thank you!